evm-adapters/src/sputnik/forked_backend/cache.rs

```diff
@@ -378,14 +379,16 @@ impl SharedBackend {
         cache: SharedCache<MemCache>,
         vicinity: MemoryVicinity,
         pin_block: Option<BlockId>,
+        runtime: Option<Runtime>,
```

Comment:
Had to pass runtime along because WebSocket connection would be closed when the runtime that it was created on drops.

Reply:
btw, i don't think `RuntimeOrHandle` was good idea here (i'm to blame) - it would be better to have clean async constructor with native spawn & wrap it in sync version.